ui/src/components/connections/Connections.tsx

```diff
@@ -0,0 +1,174 @@
+import { Avatar, Card, Divider, Row, Space, Table, Tag, Tooltip, Typography } from "antd";
+import { ColumnsType } from "antd/es/table";
+import { useCallback, useEffect, useState } from "react";
+import { useSearchParams } from "react-router-dom";
+
+import { APIError } from "src/adapters/api";
+import { Connection, Credential, getConnections } from "src/adapters/api/connections";
+import { ReactComponent as IconUsers } from "src/assets/icons/users-01.svg";
+import { ConnectionsRowDropdown } from "src/components/connections/ConnectionsRowDropdown";
+import { ErrorResult } from "src/components/shared/ErrorResult";
+import { NoResults } from "src/components/shared/NoResults";
+import { SiderLayoutContent } from "src/components/shared/SiderLayoutContent";
+import { TableCard } from "src/components/shared/TableCard";
+import { useEnvContext } from "src/contexts/env";
+import { isAbortedError, makeRequestAbortable } from "src/utils/browser";
+import { CONNECTIONS, QUERY_SEARCH_PARAM } from "src/utils/constants";
+import { AsyncTask, isAsyncTaskDataAvailable, isAsyncTaskStarting } from "src/utils/types";
+
+export function Connections() {
+  const env = useEnvContext();
+  const [connections, setConnections] = useState<AsyncTask<Connection[], APIError>>({
+    status: "pending",
+  });
+
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const queryParam = searchParams.get(QUERY_SEARCH_PARAM);
+
+  const tableContents: ColumnsType<Connection> = [
+    {
+      dataIndex: "userID",
+      ellipsis: { showTitle: false },
+      key: "type",
+      render: (userID: Connection["userID"]) => (
+        <Tooltip placement="topLeft" title={userID}>
+          <Typography.Text strong>{userID.replace(/did:\w*:\w*:\w+:/, "")}</Typography.Text>
+        </Tooltip>
+      ),
+      sorter: ({ id: a }, { id: b }) => a.localeCompare(b),
+      title: "Identifier",
+    },
+    {
+      dataIndex: "credentials",
+      key: "credentials",
+      render: (credentials: Credential[]) =>
+        [...credentials]
+          .sort((a, b) => a.attributes.type.localeCompare(b.attributes.type))
+          .map((credential) => (
+            <Typography.Text key={credential.id}>{credential.attributes.type}</Typography.Text>
+          )),
+      title: "Issued credentials",
+    },
+    {
+      dataIndex: "active",
+      key: "active",
+      render: () => <ConnectionsRowDropdown />,
+      width: 55,
```

Comment:
Why is this needed?

Reply:
Because it's not used in other places

Reply:
My question is why do we need it. If we remove it, the columns will scale according to their content, no?

Reply:
I'm going to revert to the hardcoded width because I'm seeing that the width is being evenly distributed across each column without it.

Reply:
![image](https://user-images.githubusercontent.com/19190248/228862530-1c1f3843-eb3d-4232-85f4-6423788bc7c4.png)

Reply:
If you mean to say let's consolidate this type of thing, then yes. Otherwise I don't know what you mean by this screenshot without any further context.

Reply:
I'm not going to create a constant until the moment that it's used in more than one place, don't worry it'll come soon and I'll round it to 60